nexus/reconfigurator/planning/src/mgs_updates/host_phase_1.rs

```diff
@@ -416,7 +416,9 @@ pub(super) fn try_make_update(
         // TODO-correctness Need to choose gimlet vs cosmo here! Need help from
         // tufaceous to tell us which is which.
         // https://github.com/oxidecomputer/omicron/issues/8777
-        if artifact.id.kind == ArtifactKind::HOST_PHASE_1 {
+        if artifact.id.kind == ArtifactKind::HOST_PHASE_1
+            && artifact.board.as_ref().map_or(false, |x| x == "gimlet-host")
```

Comment:
We're probably going to use this and `cosmo-host` board names all over the place right? If so, it'd be really nice to have them as constants